src/Compilers/VisualBasic/Portable/Parser/Parser.vb

```diff
@@ -2945,16 +2945,22 @@ checkNullable:
                 Dim identifierNameOpt As IdentifierNameSyntax = Nothing
                 Dim asKeywordOpt As KeywordSyntax = Nothing
 
+                ' if there is a type character or As, then this must be a name
                 If CurrentToken.Kind = SyntaxKind.IdentifierToken AndAlso
```

Comment:
>If CurrentToken.Kind = SyntaxKind.IdentifierToken AndAlso [](start = 16, length = 57)

It looks like a standalone identifier is always parsed as a type name rather than as a name of the element. In VB, with Option Strict Off, it is valid to declare a field with the following syntax: ```Dim F```. The "F" is treated as the field name, not a type. What we do here, feels somewhat inconsistent. Has LDM made an explicit decision about how we should be dealing with an ambiguity like this?